web/client/themes/default/less/geostory.less

```diff
@@ -11,7 +11,14 @@
 @ms-story-font-size-h4: floor((@ms-story-font-size-base * 1.4));
 @ms-story-font-size-h5: @ms-story-font-size-base;
 @ms-story-font-size-h6: floor((@ms-story-font-size-base * 0.85));
-
+@ms-size-full: 100%;
+@ms-size-large: 85%;
+@ms-size-medium: 60%;
+@ms-size-small: 40%;
+/* mixin for align*/
+.align-center { margin-left: auto; margin-right: auto; }
+.align-left { margin-left: 0; margin-right: auto; }
+.align-right { margin-left: auto; margin-right: 0; }
```

Comment:
We should add a specific prefix to avoid styles overrides around MapStore.
example
@ms-size-full: 100%; ->@ms-story-size-full:
...
.align-center -> .ms-story-align-center
...